conf/test_contamination.config

```diff
@@ -14,11 +14,6 @@ params {
     config_profile_name        = 'Test profile'
     config_profile_description = 'Minimal test dataset to check pipeline function with contamination filter'
 
-    // Limit resources so that this can run on GitHub Actions
```

Comment:
Don't you need the same changes as in `conf/test.config` ?

(same fore the other files below.)